### DISCUSSION TITLE
Criação de um canal do discord para comunicação dos devs

### DISCUSSION THREAD

**Discussion by swxtz** [diego3g/rsxp-2023#48] [Ideas]
Uma coisa muito comum que vejo aqui são devs perguntando se alguém já está fazendo ou já fez uma certa feature, acho uma boa criar um canal no discord onde todo mundo possa conversar mais fácil assim evitando trabalho desnecessário

**Comment by LuisCarlosDev** [comment]
Bom dia @swxtz, isso já está discussão em #30

**Comment by diego3g** [comment]
Discussão repetida, vou fechar por aqui pra não dividirmos as conversas.
